ecs-cli/integ/e2e/local_test.go

```diff
@@ -97,3 +174,40 @@ func TestECSLocal(t *testing.T) {
 		})
 	}
 }
+
+func createLocalTaskDefFile(t *testing.T) string {
+	content := `
+{
+	"containerDefinitions": [
+		{
+			"entryPoint": [
+				"sh",
+				"-c"
+			],
+			"essential": true,
+			"image": "httpd:2.4",
+			"name": "simple-test-app",
+			"portMappings": [
+				{
+				"containerPort": 80,
+				"hostPort": 80,
+				"protocol": "tcp"
+				}
+			]
+		}
+	],
+	"cpu": "256",
+	"memory": "512",
+	"networkMode": "ecs-local-network"
```

Comment:
why are we using this as the network mode? my understanding is we expect the `task-def.json` file passed into `ecs local` to be a "real" task def that could be run in production (where `ecs-local-network` doesn't exist). 
Can we change this to `bridge` instead to better model a real task def?

Reply:
Changed in most recent commit.